compendium/DeclarativeServices/src/manager/BundleLoader.cpp

```diff
@@ -163,9 +164,9 @@ namespace cppmicroservices
                 bundleBinaries.lock()->emplace(bundleLoc, handle);
             }
 
-            const std::string symbolName = std::regex_replace(compName, std::regex("::"), "_");
-            const std::string newInstanceFuncName("NewInstance_" + symbolName);
-            const std::string deleteInstanceFuncName("DeleteInstance_" + symbolName);
+            std::string const symbolName = std::regex_replace(compName, std::regex("::"), "_");
```

Comment:
Regex replace has the reputation of being very slow. Especially for simple replacements.
As a suggestion a simple replace lambda loop could be added here. For estimated 10x improvement.